moveit_ros/planning/planning_request_adapters_plugin_description.xml

```diff
@@ -47,4 +47,9 @@
     </description>
   </class>
 
+  <class name="default_planner_request_adapters/PublishIntermediateTrajectory" type="default_planner_request_adapters::PublishIntermediateTrajectory" base_class_type="planning_request_adapter::PlanningRequestAdapter">
+    <description>
```

Comment:
Could you add a description about how to use this, and a note about not being able to have more than one per pipeline?

Reply:
Done.